beam-core/test/Database/Beam/Test/SQL.hs

```diff
@@ -1366,3 +1369,52 @@ gh70OrderByInFirstJoinCausesIncorrectProjection =
                    , selectLimit = Nothing , selectOffset = Nothing }
 
       actual @?= expected
+
+-- | Test ExprAll
+
```

Comment:
nit: here and below, haddocks should be adjacent to declaration

Reply:
It does look weird, doesn't it? I just followed the example of earlier definitions in this file, like https://github.com/haskell-beam/beam/pull/590/files#diff-f4e085f7289c80ab762596354429a86fc78598f1bd0ff87557d56527a392384fR1173